codes.json

```diff
@@ -732,6 +732,14 @@
       "en": "MapMonitoring status must be either enabled or disabled (case insensitive)."
     }
   },
+  "InvalidFormat": {
```

Comment:
can you add more detail about what's invalid_format is? is it invalid_XLS_format or what? Also I think we have invalid csv format error message above or you can reuse if it's a general error message